packages/shared/scope-hoisting/src/concat.js

```diff
@@ -11,11 +11,17 @@ import fs from 'fs';
 import nullthrows from 'nullthrows';
 import {PromiseQueue} from '@parcel/utils';
 
+const EslintCommentsRegExp = /^\s*\/\/\s*eslint.*?$|^\s*\/\*\s*eslint.*?\s*\*\/$/gm;
```

Comment:
this variable name should probably start with a lowercase letter to stay consistent with the rest of the codebase